front/src/modules/timetableItem/components/Timetable/types.ts

```diff
@@ -20,30 +20,32 @@ export type TrainCategoryFilter = 'all' | 'noCategory' | TrainMainCategory;
 
 type SimulationSummaryResultSuccess = Extract<SimulationSummaryResult, { status: 'success' }>;
 
-type TimetableItemWithSummaries = Omit<
+export type SimulationSummary =
+  | { isValid: false; invalidReason: InvalidReason }
+  | {
+      isValid: true;
+      duration: Duration /** Travel time */;
```

Comment:
nit : would be nicer like that so we can have the comment when hovering the property in other files
```suggestion
      /** Travel time */
      duration: Duration;
```

Reply:
Done